homeassistant/components/husqvarna_automower/number.py

```diff
@@ -32,6 +32,15 @@ class AutomowerNumberEntityDescription(NumberEntityDescription):
     set_value_fn: Callable[[AutomowerSession, str, float], Awaitable[Any]]
 
 
+@callback
+def _async_get_cutting_height(data: MowerAttributes) -> int:
```

Comment:
Why async in the name if it's not async?

Reply:
It hass a `callback` decorator indicating it can be safely called from an async method